tests/default_vars.sh

```diff
@@ -307,7 +307,7 @@ export_fv3 ()
 {
 export FV3=true
 export S2S=false
-export DATM=false
+export DATM_CDEPS=false
```

Comment:
Again, I was thinking that we were plaining to preserve the DATM already exist in UFS model and I made changes based on it in HAFS side. Maybe the plans are changed but I don't have that information and it seems there is a lack of communication in here.